conf.yaml

```diff
@@ -476,6 +477,11 @@ contents:
                     repo:   elasticsearch
                     path:   client
                     exclude_branches:   [ 5.5, 5.4, 5.3, 5.2, 5.1, 5.0 ]
+                  -
```

Comment:
If we want to keep this as part of the "Java Client" book, this is fine, but I'd also be fine splitting this into a separate book, since it's now a separate project in a separate repo.